Sources/MongoSwift/ReadConcern.swift

```diff
@@ -62,7 +62,7 @@ public struct ReadConcern: Codable {
 
     // Initializes a new `ReadConcern` with the same level as the provided `mongoc_read_concern_t`.
     // The caller is responsible for freeing the original `mongoc_read_concern_t`.
-    internal init(from readConcern: OpaquePointer) {
+    internal init(copying readConcern: OpaquePointer) {
```

Comment:
`copying` seemed more descriptive and consistent with our other similar internal APIs